src/modules/ce/components/AssignContactFormItem.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback, useMemo } from 'react';
+import FormSelect from '@/modules/form/components/FormSelect';
+import {
+  CeReferralSwimlaneFieldsFragment,
+  PickListOption,
+  PickListType,
+  useGetPickListQuery,
+} from '@/types/gqlTypes';
+
+interface Props {
+  swimlane: CeReferralSwimlaneFieldsFragment;
+  users: string[];
+  setUsers: (userIds: string[]) => void;
+}
+const AssignContactFormItem: React.FC<Props> = ({
+  swimlane,
+  users,
+  setUsers,
+}) => {
+  const {
+    data: { pickList: staffPickList } = {},
+    loading: staffPickListLoading,
+    error: staffPickListError,
+  } = useGetPickListQuery({
+    variables: {
+      pickListType: PickListType.Users,
```

Comment:
Thinking ahead: Need a `TODO` here for permissions ticket (or thereafter) to introduce a new picklist (like `ELIGIBLE_REFERRAL_PARTICIPANT_USERS`) that respects the referral participation permission(s). We will need to pass a `projectId` to the picklist query here too, when we implement that.